app/uploaders/alert_image_uploader.rb

```diff
@@ -10,40 +12,47 @@ def base_store_dir
     "uploads/#{stolen_record.class.to_s[0, 2]}"
   end
 
-  def filename
-    return if stolen_record.alert_image.blank?
-    file = File.basename(stolen_record.alert_image.path, ".*").chomp("-alert")
-    "#{file}-alert.jpg"
-  end
-
   def extension_white_list
     %w(jpg jpeg gif png tiff tif)
   end
 
-  process :generate_alert_image
+  process :fix_exif_rotation
+  process :strip
   process convert: "jpg"
 
-  alias stolen_record model
-  delegate :bike, to: :model
+  version :landscape do
+    process :generate_landscape
+    process resize_to_fill: [1200, 630]
+  end
 
-  def bike_url
-    "bikeindex.org/bikes/#{bike.id}"
+  version :square do
+    process :generate_square
+    process resize_to_fill: [1200, 1200]
   end
 
-  def bike_location
-    if stolen_record.address_location.present?
-      stolen_record.address_location
-    else
-      bike.registration_location
+  def generate_landscape
+    Rails.logger.info "Processing landscape #{current_path}"
```

Comment:
I'm somewhat suspicious of logging statements, because it's difficult to know when it's ok to delete them.

I understand the utility of these, because this is background processed and otherwise it might be difficult to get visibility - but - would you create a separate PR once this is merged, that removes them? So we know it's something that should happen and it's a streamlined process

Reply:
No prob -- I won't keep these around too long, just want to get some insight into that empty body exception.

Reply:
Thanks!